components/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/utils/SCIMCommonUtils.java

```diff
@@ -221,7 +222,12 @@ public static void setAdminSCIMAttributes() throws UserStoreException {
                     (UserStoreManager) SCIMCommonComponentHolder.getRealmService().
                             getTenantUserRealm(superTenantId).getUserStoreManager();
 
-            if (userStoreManager.isSCIMEnabled()) {
+            IdentityEventListenerConfig identityEventListenerConfig = IdentityUtil.readEventListenerProperty(
+                    "org.wso2.carbon.user.core.listener.UserOperationEventListener",
+                    "org.wso2.carbon.identity.scim.common.listener.SCIMUserOperationListener"
+            );
+
+            if (identityEventListenerConfig.getEnable().equals("true") && userStoreManager.isSCIMEnabled()) {
```

Comment:
let's switch the arguments of equals as
"true".equals(identityEventListenerConfig.getEnable())

This will handle possible NPEs

Reply:
How about using `Boolean.parse` in here?

Reply:
Fixed in commit adfe3f06759f3cb84295bcd7566681526c5171a6